src/framework/ui/internal/uiconfiguration.cpp

```diff
@@ -67,6 +67,8 @@ static const QMap<ThemeStyleKey, QVariant> LIGHT_THEME_VALUES {
     { FONT_SECONDARY_COLOR, "#FFFFFF" },
     { LINK_COLOR, "#0B69BF" },
     { FOCUS_COLOR, "#75507b" },
+    { MUSESCORE_TEXT_LOGO_COLOR, "#0065C3" },
+    { AUDIOCOM_LOGO_COLOR, "#000000" },
```

Comment:
We need to solve this problem in the model/service classes because adding these fields only solves your problem, these fields will not be used anywhere else